src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/GappedAlignmentSplitterUnitTest.java

```diff
@@ -25,106 +25,106 @@ public void testCompactifyNeighboringSoftClippings() {
     public void testGappedAlignmentBreaker_OneInsertion() {
 
         final Cigar cigar = TextCigarCodec.decode("56S27M15I32M21S");
-        final AlignmentInterval alignmentInterval = new AlignmentInterval(new SimpleInterval("1", 100, 158), 57, 130, cigar, true, 60, 0, 100, false);
+        final AlignmentInterval alignmentInterval = new AlignmentInterval(new SimpleInterval("1", 100, 158), 57, 130, cigar, true, 60, 0, 100, false, false);
 
         final List<AlignmentInterval> generatedARList = Utils.stream(GappedAlignmentSplitter.split(alignmentInterval, 1, cigar.getReadLength())).collect(Collectors.toList());
         Assert.assertEquals(generatedARList.size(), 2);
-        Assert.assertEquals(generatedARList.get(0), new AlignmentInterval(new SimpleInterval("1", 100, 126), 57, 83, TextCigarCodec.decode("56S27M68S"), true, 60, StructuralVariationDiscoveryArgumentCollection.DiscoverVariantsFromContigsAlignmentsSparkArgumentCollection.MISSING_NM, 100, true));
-        Assert.assertEquals(generatedARList.get(1), new AlignmentInterval(new SimpleInterval("1", 127, 158), 99, 130, TextCigarCodec.decode("98S32M21S"), true, 60, StructuralVariationDiscoveryArgumentCollection.DiscoverVariantsFromContigsAlignmentsSparkArgumentCollection.MISSING_NM, 100, true));
+        Assert.assertEquals(generatedARList.get(0), new AlignmentInterval(new SimpleInterval("1", 100, 126), 57, 83, TextCigarCodec.decode("56S27M68S"), true, 60, StructuralVariationDiscoveryArgumentCollection.DiscoverVariantsFromContigsAlignmentsSparkArgumentCollection.MISSING_NM, 100, true, false));
```

Comment:
You might want to split up some of these long lines for readability while you're in here.